### PR TITLE
[KARAF-2511] Add karaf-log-appender-example

### PR DESCRIPTION
**Pull request by jbonofre**

### CODE REVIEW

File: examples/karaf-log-appender-example/karaf-log-appender-example-core/pom.xml [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+    <artifactId>karaf-log-appender-example-core</artifactId>
+    <name>Apache Karaf :: Examples :: Log Appender :: Core</name>
+    <packaging>bundle</packaging>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.ops4j.pax.logging</groupId>
+            <artifactId>pax-logging-service</artifactId>
+            <version>1.8.5</version>
```
Comment (fpapon) [inline examples/karaf-log-appender-example/karaf-log-appender-example-core/pom.xml]: We should use the Karaf parent version.
Please remove the version tag.

**Review by fpapon — COMMENTED** [review]
Please update the core pom.xml

**Comment by jbonofre** [comment]
Updated/rebased, ready for a new review round.

**Comment by fpapon** [comment]
LGTM
Can you squash please ?

**Comment by jbonofre** [comment]
squashed

**Pull request merged**